versioning/BRM/extentmap.cpp

```diff
@@ -347,8 +347,16 @@ int ExtentMap::_markInvalid(const LBID_t lbid, const execplan::CalpontSystemCata
 
                 if (isUnsigned(colDataType))
                 {
-                    fExtentMap[i].partition.cprange.bigLoVal = -1;
-                    fExtentMap[i].partition.cprange.bigHiVal = 0;
+                    if (fExtentMap[i].colWid != datatypes::MAXDECIMALWIDTH)
+                    {
+                        fExtentMap[i].partition.cprange.loVal = numeric_limits<uint64_t>::max();
+                        fExtentMap[i].partition.cprange.hiVal = numeric_limits<uint64_t>::min();
+                    }
+                    else
+                    {
+                        fExtentMap[i].partition.cprange.bigLoVal = -1; // XXX: unsigned wide decimals do not exceed rang of signed wide decimals.
+                        fExtentMap[i].partition.cprange.bigHiVal = 0;
+                    }
```

Comment:
I am curious about this change. Shouldn't setting the below suffice for both narrow and wide cases:

```
fExtentMap[i].partition.cprange.bigLoVal = -1;
fExtentMap[i].partition.cprange.bigHiVal = 0;
```